shortcircuit-api/src/main/java/com/powsybl/shortcircuit/ShortCircuitConstants.java

```diff
@@ -14,7 +14,23 @@ public final class ShortCircuitConstants {
     private ShortCircuitConstants() {
     }
 
-    public static final boolean SUBTRANS_STUDY = false;
+    public enum StudyType {
+        SYSTEMATIC_STUDY,
+        SELECTIVE_STUDY,
+    }
+
+    public enum SelectiveStudyType {
+        BUS_STUDY,
+        BRANCH_STUDY,
+    }
 
+    public enum ImpedanceConnection {
+        SERIES,
+        PARALLEL,
+    }
+
+    public static final boolean SUBTRANS_STUDY = false;
     public static final boolean WITH_FEEDER_RESULT = true;
```

Comment:
Shouldn't simplify this at a first implementation:
- if SYSTEMATIC_STUDY, we don't display non-mandatory data (all bus voltages, feeder)
- if SELECTIVE_STUDY, then everything is provided (maybe check user practices...)

Reply:
As we will have two different `run`, one for systematic study, one for selective study, this parameter could be removed.